knowledge_base/private_wheel_packages/README.md

```diff
@@ -0,0 +1,41 @@
+# Private wheel packages
+
+This example demonstrates how to use a private wheel package from a job in a Databricks Asset Bundle.
+
+## Prerequisites
+
+* Databricks CLI v0.235.0 or above
+* Python 3.10 or above
+
+# Usage
+
+Typically, private wheels are hosted in a private repository and are not built inline.
+To emulate this, we will build a local copy of an example wheel that does not exist on PyPI and copy it to the `dist` directory.
```

Comment:
https://github.com/databricks/bundle-examples/pull/47/files



So the goal here is to show that you can distribute privately built wheel files by putting them in dist?

I think this example would be more self-explanatory by either adding a binary wheel to the repository or by providing a link to a wheel file that customers can download themselves. Then the example can just focus on wheel distribution and doesn't need all the extra files and explanation for building a wheel.

Btw note that for notebooks, https://docs.databricks.com/en/libraries/notebooks-python-libraries.html offers an alternative for working with private repositories.

Reply:
Agreed with Lennart here, building wheel locally confused me as well, just calling out downloading and putting wheel into `dist` folder should be enough

Reply:
Good point. I updated the template to download a wheel from PyPI to simulate a private wheel.